test/test_torch.py

```diff
@@ -3297,6 +3297,56 @@ def test_scatter_add_bool(self, device):
                                             [False, True, False, True, False],
                                             [True, False, True, False, True]], device=device))
 
+    # FIXME: move to test_scatter_gather_ops.py
```

Comment:
Sorry about that, previously this test was in the class `TestTorch` which does not allow parameterization with @dtypes because it is not instantiated with `instantiate_device_type_tests(TestTorch, globals())`. Moved this here into the class `TestTorchDeviceType` which allows parameterization by @dtypes. Is there another way around this?